### PR TITLE
feat(US156): create api health check

### PR DESCRIPTION
**Pull request by BeTapLamCoder**

### CODE REVIEW

File: src/services/healthService.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+
+import { checkDatabase } from "./databaseService";
+
+export async function checkExternalAPI(): Promise<string> {
```
Comment (QUanlE79) [inline src/services/healthService.ts:5]: what is purpose of this function

File: src/services/healthService.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    try {
+        await axios.get("https://jsonplaceholder.typicode.com/posts");
+        return "connected";
+    } catch (err) {
+        console.error("External API error:", err.message);
+        return "disconnected";
+    }
+}
+
+export async function getHealthStatus(startTime: number) {
```
Comment (QUanlE79) [inline src/services/healthService.ts:15]: what is purpose of this function? write comments in code if you could

File: src/services/databaseService.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import db from "../database/migarations/database";
```
Comment (QUanlE79) [inline src/services/databaseService.ts:1]: service file should in format <servicename>.service.ts

File: src/database/migarations/database.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import knex from 'knex';
```
Comment (QUanlE79) [inline src/database/migarations/database.ts:1]: this folder is used for db migration incase db is down. it should contain .sql file. we can move db connection to utils folder

**Comment by QUanlE79** [comment]
You understand the ticket in the wrong way. I created comment TODO in healthRoute, rename it to health.route.ts and write your code there. Just simple logic return status code 200 with message "Server is healthy" or something related. If user can not connect to server, http request auto return 500, no need to handle logic here.

**Comment by BeTapLamCoder** [comment]
I will fix it soon

**Pull request closed without merging**